src/Component/Metadata/HttpOperation.php

```diff
@@ -34,8 +34,12 @@ public function __construct(
         ?string $grid = null,
         ?bool $read = null,
         ?bool $write = null,
+        ?bool $validate = null,
         ?string $formType = null,
         ?array $formOptions = null,
+        ?string $stateMachineComponent = null,
+        ?string $stateMachineTransition = null,
+        ?string $stateMachineGraph = null,
```

Comment:
This should not be generic operation in my opinion. Only creating and updating should take advantage of it

Reply:
Hum indeed, so maybe with an interface to know if state machine is applicable.